lib/annotate_rb/parser.rb

```diff
@@ -127,6 +127,15 @@ def add_wrapper_options_to_parser(option_parser)
         @options[:wrapper_open] = wrapper_open
       end
 
+      option_parser.on("--classes-default-to-s class",
+        "Custom classes to be represented with `to_s`, may be used multiple times") do |klass|
+        @options[:classes_default_to_s] = if @options[:classes_default_to_s].present?
+          [@options[:classes_default_to_s], ::Object.const_get(klass)]
```

Comment:
When there's multiple instances of this CLI option it will keep wrapping the previous value:

`DATABASE_ADAPTER=sqlite3 be annotaterb models --classes-default-to-s Integer --classes-default-to-s String`

```ruby
[1] pry(#<AnnotateRb::Runner>)> @options
=> #<AnnotateRb::Options:0x0000000143f19db8
 @options=
  {:target_action=>:do_annotations,
   :exit=>false,
   :original_args=>["models", "--classes-default-to-s", "Integer", "--classes-default-to-s", "String"],
   :classes_default_to_s=>[[Integer], String],
```

Notice that `Integer` is wrapped in an array. We may want to append here.

Reply:
Thanks, missed it completely. I used splat operator (*) because I prefer to avoid mutations if possible, let me note if you think appending is better in this case

Reply:
Splat operator also works 👍